src/oc-id/config/application.rb

```diff
@@ -33,7 +33,7 @@ class Application < Rails::Application
       Doorkeeper::AuthorizationsController.layout 'application'
       Doorkeeper::AuthorizedApplicationsController.layout 'application'
     end
-
+    config.public_file_server.enabled = true
```

Comment:
is this required for caching? nginx does caching for server,  could you elaborate a bit more about why this needs to be set?

Reply:
I have remove this.